pkg/icingadb/db.go

```diff
@@ -110,7 +110,14 @@ func (db *DB) CheckSchema(ctx context.Context) error {
 
 	var version uint16
 
-	err := db.QueryRowxContext(ctx, "SELECT version FROM icingadb_schema ORDER BY id DESC LIMIT 1").Scan(&version)
+	err := retry.WithBackoff(
+		ctx,
+		func(ctx context.Context) error {
+			return db.QueryRowxContext(ctx, "SELECT version FROM icingadb_schema ORDER BY id DESC LIMIT 1").Scan(&version)
+		},
+		shouldRetry,
+		backoff.NewExponentialWithJitter(128*time.Millisecond, 1*time.Minute),
+		db.getDefaultRetrySettings())
```

Comment:
Shouldn't this now wrap the error using `CantPerformQuery()` as the logging in `db.getDefaultRetrySettings()` relies on this to at least provide some context for where the error happened?